docs/fundamentals/executor/executor-built-in-features.md

```diff
@@ -195,7 +195,7 @@ different purposes.
 
 - **`.metas` are statically defined.** "Static" means, e.g. from hard-coded value in the code, from a YAML file.
 - **`.runtime_args` are dynamically determined during runtime.** Means that you don't know the value before running
-  the `Executor`, e.g. `pea_id`, `replicas`, `replica_id`. Those values are often related to the system/network
+  the `Executor`, e.g. `pea_id`, `replicas`. Those values are often related to the system/network
```

Comment:
can u please change pea_id to shard_id? I forgot to change in my old PR

Reply:
Sure thing